apps/core-snowpack/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "core-snowpack",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "snowpack dev",
+    "build": "snowpack build"
+  },
+  "dependencies": {
+    "normalize.css": "^8.0.1",
```

Comment:
same question as above about npm dependency vs consuming from cdn.

Reply:
This demo using the cdn for the css should be temporary. Waiting for a bugfix on snowpack where it wasnt importing css from node_modules properly.